donut/modules/groups/routes.py

```diff
@@ -35,9 +35,3 @@ def get_groups(group_id):
 def get_group_members(group_id):
     """GET /1/groups/<int:group_id>/"""
     return jsonify(helpers.get_members_by_group(group_id))
-
-
-@blueprint.route("/1/positions/<int:pos_id>/")
```

Comment:
@RobertEng did you intend for this to be removed? I think it's breaking the test build right now.

Reply:
I can take a look at them tomorrow. I'll go back to one of the branches and see if I can revert these endpoints to what they're supposed to be.